Rakefile

```diff
@@ -106,7 +106,9 @@ end
 
 desc 'Build and preview the site locally in development mode'
 task :preview, [:profile] => :check do |task, args|
-  run_awestruct "-P #{args[:profile] || 'development'} -a -s --force -q --auto --livereload -b 0.0.0.0"
+  profile = args[:profile] || 'development'
+  awestruct_running_port = awestruct_port(profile)
+  run_awestruct "-P #{profile} -a -s --force -q --auto --livereload -b 0.0.0.0 -p #{awestruct_running_port}"
```

Comment:
This is not a change I am bringing in. Ok if we look at this in a separate change outside of this PR?